src/room.ts

```diff
@@ -353,15 +347,15 @@ export class Room extends EventEmitter implements Sayable {
 
   // should be deprecated
   public nick(contact: Contact): string {
-    log.warn('Room', 'nick(Contact) DEPRECATED, use name(Contact) instead.')
+    log.warn('Room', 'nick(Contact) DEPRECATED, use alias(Contact) instead.')
     return this.alias(contact)
   }
 
   public alias(contact: Contact): string {
-    if (!this.obj || !this.obj.nameMap) {
+    if (!this.obj) {
       return ''
     }
-    return this.obj.roomAliasMap[contact.id] || this.obj.nameMap[contact.id]
+    return this.obj.aliasMap[contact.id]
```

Comment:
if no `aliasMap[contact.id]` here, it should return undefiend. Actually, I have deal alias in aliasMap as follows:
```ts
private parseMap(memberList: RoomRawMember[], parseContent: NameType): Map<string, string> {
    const mapList: Map<string, string> = new Map<string, string>()
    if (memberList && memberList.map) {
      memberList.forEach(member => {
        let tmpName: string
        let contact = Contact.load(member.UserName)
        switch (parseContent) {
          case 'name':
            tmpName = contact.alias() || contact.name()
            break
          case 'alias':
            tmpName = member.DisplayName || contact.name()
            break
          default:
            throw new Error('parseMap failed, member not found')
        }
```

if no `DisplayName`, this code `tmpName = member.DisplayName || contact.name()` will get `contact.name()`   

So, no need to add `this.obj.nameMap[contact.id]` here, what do you think about it?